src/core_modules/capture-core/components/Pages/EnrollmentAddEvent/EnrollmentAddEventPage.component.js

```diff
@@ -60,8 +73,26 @@ const EnrollmentAddEventPagePain = ({
                     }
                     noncollapsible
                 >
-                    [event details]
+                    <div data-test="add-event-form">
```

Comment:
Hey, 
Here, I think you should use `Widget` and moved all the form-related logic inside it. The same pattern is used on the [Enrollment](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js#L53) page. The page only handles the context and passes it to different Widgets. You can also look at `WidgetEventEdit` for some ideas. 
Thanks!

Reply:
hey @simonadomnisoru, thank you for the comment, I moved the form to `WidgetAddEvent`